pkg/core/db.go

```diff
@@ -23,6 +26,23 @@ type NoopDB struct {
 
 // SetUp initializes the database.
 func (NoopDB) SetUp(ctx context.Context, nodes []cluster.Node, node cluster.Node) error {
+	prepareSQL := fixture.Context.TiDBClusterConfig.PrepareSQL
+	if len(prepareSQL) > 0 && node.Component == cluster.TiDB {
+		db, err := sql.Open("mysql", fmt.Sprintf("root@tcp(%s:%d)/test", node.IP, node.Port))
```

Comment:
I suggest adding `multiStatements=true` To support multiple statements.

```suggestion
		db, err := sql.Open("mysql", fmt.Sprintf("root@tcp(%s:%d)/test?multiStatements=true", node.IP, node.Port))
```